docs/data/sources/google-tag-manager.md

```diff
@@ -1,12 +1,49 @@
 ---
-title: Google Tag Manager
+title: Google Tag Manager Template Quickstart Guide
 description: Official Google Tag Manager template for Amplitude Analytics for data collection.
 ---
 
-Use the official Google Tag Manager template for Amplitude Analytics for data collection. 
+Use the official Amplitude Analytics Google Tag Manager templates to collect data. Choose a template to visit the documentation and tutorial video.  
 
-|Template|Description|
-|---|-----------|
-|[Client-side tag](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-browser-sdk-gtm-template)|The tag uses the Amplitude Marketing Analytics SDK for data collection.|
-|[Client-side tag (legacy)](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-gtm-template)| The tag uses the Amplitude JavaScript SDK for data collection. |
-|[Server-side tag](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-server-gtm-template)| The tag uses the Amplitude HTTP API for data collection. |
\ No newline at end of file
+|Template|Type|Description|
+|---|---|-----------|
+|[Amplitude Analytics Browser SDK](../sources/google-tag-manager-client/)|Client-side|The tag uses the Amplitude Marketing Analytics SDK for data collection.|
+|[Amplitude Analytics](../sources/google-tag-manager-server/)|Server-side|The tag uses the Amplitude HTTP V2 API for data collection. |
+|[Amplitude Analytics Legacy](../sources/google-tag-manager-client-legacy/)|Client-side|The tag uses the Amplitude JavaScript SDK for data collection. |
```

Comment:
Also, do we want to change to `maintenance` to be align with https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/pull/589?

Reply:
I prefer keep using "legacy" here because the name of the template is [Amplitude Analytics Legacy](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-gtm-template). Although we could change the name but that change may cause name confusion to customers who are using it.

Open to discussion if you have any thought!